graylog2-server/src/main/java/org/graylog2/migrations/V20161216123500_DefaultIndexSetMigration.java

```diff
@@ -73,7 +73,7 @@ public void upgrade() {
             final String analyzer = elasticsearchConfiguration.getAnalyzer();
             final IndexSetConfig updatedConfig = indexSetConfig.toBuilder()
                     .indexAnalyzer(analyzer)
-                    .indexTemplateName(elasticsearchConfiguration.getTemplateName())
+                    .indexTemplateName(indexSetConfig.indexPrefix() + "-template")
```

Comment:
No, it simply let users decide how to name the Graylog index template. If they needed modifications of the mapping, they could override them with a custom index template: http://docs.graylog.org/en/2.1/pages/configuration/elasticsearch.html#custom-index-mappings